data-pipeline/tests/unit/pre_processing/test_mappings.py

```diff
@@ -297,13 +297,13 @@ def test_map_school_website(url, expected):
 @pytest.mark.parametrize(
     "pfi,expected",
     [
-        ("part of PFI", "PFI School"),
-        ("not PFI", "Non-PFI school"),
-        ("", "Non-PFI school"),
+        (1, "PFI School"),
+        (0, "Non-PFI school"),
+        (-1, "PFI School"),
```

Comment:
might be a silly question, but if we're just checking the integers, why not use the absolute and remove line 302?